presto-main/src/main/java/io/prestosql/sql/planner/LocalExecutionPlanner.java

```diff
@@ -1181,6 +1181,18 @@ private PhysicalOperation visitScanFilterAndProject(
                     channel++;
                 }
             }
+            //TODO: This is a simple hack, it will be replaced when we add ability to push down sampling into connectors.
+            // SYSTEM sampling is performed in the coordinator by dropping some random splits so process only for it.
```

Comment:
I would update the second part of the comment, to say `... some random splits, so the SamplingNode can be skipped here`

Reply:
Updated the second part of the comment